mozci/task.py

```diff
@@ -303,7 +307,8 @@ def backfill(self, push):
                 "taskGroupId": decision_task.id,
                 "input": {
                     "times": 5
-                    if self.classification == "not classified" or self.classification in INTERMITTENT_CLASSES
+                    if self.classification == "not classified"
+                    or self.classification in INTERMITTENT_CLASSES
```

Comment:
Nit: could you revert this unnecessary change?

Reply:
Weird that it doesn't fail on master :S
Anyway, I've run it locally and pushed the reformatted code to master, so you can rebase.